libs/orders/src/lib/components/order-feedback/order-feedback.tsx

```diff
@@ -113,6 +74,27 @@ export const OrderFeedback = ({ transaction, order }: OrderFeedbackProps) => {
           </div>
         )}
       </div>
+      {orderRejectionReason && (
+        <div>
+          <p className={labelClass}>{t(`Reason`)}</p>
+          <p data-testid="error-reason">{t(orderRejectionReason)}</p>
+        </div>
+      )}
     </div>
   );
 };
+
+const rejectionReason = (
+  order: OrderEvent_busEvents_event_Order
+): string | null => {
+  switch (order.status) {
+    case OrderStatus.Stopped:
+      return t(
+        `The network could not fill the ${order.timeInForce} order and it has been stopped`
+      );
+    case OrderStatus.Rejected:
+      return order.rejectionReason && t(formatLabel(order.rejectionReason));
```

Comment:
reason coming from the backend only when the order is Rejected. we could still add other messages that can appear when the order is Cancelled or Parked.